cves/kernel/CVE-2021-38300.yml

```
@@ -167,8 +174,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: This bug could've been cached earlier with a check for out-of-range branches
```

```suggestion
  note: This bug could've been caught earlier with a check for out-of-range branches
```
minor wording change suggestion (`catched` -> `caught`)